charts/backstage/values.yaml

```diff
@@ -65,10 +65,9 @@ ingress:
   className: ""
 
   ## @param ingress.annotations Additional annotations for the Ingress resource
-  annotations: '{
-    "nginx.ingress.kubernetes.io/rewrite-target": "/",
+  annotations:
+    "nginx.ingress.kubernetes.io/rewrite-target": "/"
```

Comment:
Do we even want to provide these as default? How about just providing an empty annotation object `annotations: {}`? I'm just thinking about the possible impact that providing these two annotations will have on users.

Reply:
I think you may be right. @vinzscam WDYT?

Reply:
yeah good point @ChrisJBurns

Reply:
Ok, changing the PR to remove default annotations + treat them as a dict instead of a string than. :slightly_smiling_face: